omod-2.0/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/converter/openmrs2_0/VisitsConfigurationConverter2_0.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.converter.openmrs2_0;
+
+import org.openmrs.VisitType;
+import org.openmrs.annotation.Handler;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.api.RestService;
+import org.openmrs.module.webservices.rest.web.representation.FullRepresentation;
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingConverter;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.response.ConversionException;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_9.VisitTypeResource1_9;
+import org.openmrs.module.webservices.rest.web.v1_0.wrapper.VisitsConfiguration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Handler(supports = VisitsConfiguration.class, order = 0)
+public class VisitsConfigurationConverter2_0 extends BaseDelegatingConverter<VisitsConfiguration> {
+
+	@Override
+	public DelegatingResourceDescription getRepresentationDescription(Representation rep) {
+		DelegatingResourceDescription description = new DelegatingResourceDescription();
+		description.addProperty("visitEncounterHandler");
+		description.addProperty("enableVisits");
+		description.addProperty("autoCloseVisitsTaskStarted");
```

Comment:
Shouldn't this be `startAutoCloseVisitsTask`?

Reply:
How about the encounter visits assignment handler?

Reply:
@dkayiwa If we're retrieving a configuration then `startAutoCloseVisitsTask` doesn't make sense as we want to know whether this task is started or not. It might make sense to use different wording on POST but then we would have to introduce two objects, one for retrieving and one for updating.

Reply:
> How about the encounter visits assignment handler?

Do you want me to change the name of that field to `encounterVisitsAssignmentHandler`?

Reply:
Looks better.

Reply:
> If we're retrieving a configuration then `startAutoCloseVisitsTask` doesn't make sense as we want to know whether this task is started or not. It might make sense to use different wording on POST but then we would have to introduce two objects, one for retrieving and one for updating.

What is our convention else where?

Reply:
Honestly I couldn't find any other examples in REST WS module. However the LegacyUI uses the following name for this field: "Start auto close visits task ". So I guess I could change that to `startAutoCloseVisitsTask`

Reply:
Resolved here: https://github.com/openmrs/openmrs-module-webservices.rest/pull/489/commits/cb0f562a61bd955af39b1bbd20ee8abc4cbbb42c